pulp_ansible/app/viewsets.py

```diff
@@ -376,6 +391,16 @@ class Meta:
             "metadata_sha256": ["exact", "in"],
         }
 
+    def no_op(self, queryset, name, value):
+        return queryset
+
+    def has_permissions(self, queryset, name, value):
+        perms = self.request.query_params.getlist(name)
```

Comment:
How is that different from `value`?

Reply:
Looks like it needs to be taken using getlist() so it returns a Python list type.